REQUESTLOG.MARCJOSEPH.md

```diff
@@ -0,0 +1,15 @@
+# How to pull request
+1. (Date & Time of Request) i.e "10/23/2017 5:45 pm" (Passenger & Destination in Question) "Regina to Margerita's Pizzeria" add date and and estimated time of pickup "at/for 10:45 pm 10/24/2017" add the request bracket and put "| requested []" the parent then approves or denies the request if denial is given a reason has to be given and can be up for a dispute a reason can look like this "reason: too late over curfew & short notice."
+
+## Pull request approval 
+1. As so "10/23/2017 5:45 pm Regina to Margerita's Pizzeria at 10:45 pm 10/24/2017| requested []" 
+- for approval: "10/23/2017 5:45 pm Regina to Margerita's Pizzeria at 10:45 pm 10/24/2017| request [APPROVED]"
+- for denial: "10/23/2017 5:454 pm Regina to Margerita's Pizzeria at 10:45 pm 10/2024/2017| request [DENIED] reason: too late over curfew & short notice" 
+
+## Disputes
+1. Disputes can be over call, over text, or in person
```

Comment:
but final results are recorded in the PR - helps everyone including the parent learn